src/event.js

```diff
@@ -187,17 +188,18 @@ jQuery.event = {
 
 				// Only use addEventListener if the special events handler returns false
 				if ( !special.setup ||
-					special.setup.call( elem, data, namespaces, eventHandle ) === false ) {
+					special.setup.call( elem, data, namespaces, handleObj ) === false ) {
 
 					if ( elem.addEventListener ) {
-						elem.addEventListener( type, eventHandle );
+						elem.addEventListener( type, handleObj );
```

Comment:
Hmm, I'll definitely look into that. Seems like a LOT of stuff should have broken.